### PR TITLE
Add karma and jasmine unit testing to lib-jitsi-meet.  also add a ver…

### PR DESCRIPTION
**Pull request by bbaldino**
…y basic example test for SDPUtil

### CODE REVIEW

**Comment by bgrozev** [comment]
jenkins add to whitelist

File: .jshintignore [hunk 1]
```diff
@@ -30,3 +31,4 @@ JitsiParticipant.js
 JitsiMeetJS.js
 JitsiConnection.js
 JitsiConference.js
+modules/xmpp/SampleSdpStrings.js
```
Comment (hristoterezov) [inline .jshintignore]: Is this added by mistake?

Comment (bbaldino) [inline .jshintignore]: yes it is...good catch.  this is from the testing i have on my other branch

File: .jshintignore [hunk 2]
```diff
@@ -3,6 +3,7 @@ libs
 lib-jitsi-meet.js
 lib-jitsi-meet.min.js
 lib-jitsi-meet.js.map
+karma.conf.js
```
Comment (hristoterezov) [inline .jshintignore]: Why do we need to exclude karma.conf.js?

webpack.config.js is not excluded for example, what is the difference?

Comment (bbaldino) [inline .jshintignore]: oops.  originally there were some symbol complaints there but now looks like it's just line lengths.  i'll fix that.

**Pull request merged**